website/docs/dbt-cli/install/docker.md

```diff
@@ -3,8 +3,117 @@ title: "Use Docker to install dbt"
 description: "You can use Docker to install dbt and adapter plugins from the command line."
 ---
 
-By v1.0.0, dbt Core and all adapter plugins maintained by dbt Labs will be available as official Docker images,
-and available from a public registry.
-We recommend you use Docker to install in production because it includes dbt Core and all of its dependencies. You might might also use a Docker to install and develop locally if you don't have your python environment set up.
+You will find dbt Core and all adapter plugins maintained by dbt Labs available as [Docker](https://docs.docker.com/) images. We recommend you use Docker to install dbt in production because it includes dbt Core, one or more database adapters, and all dependencies. You can use Docker to install and develop locally if you don't have a python environment set up.
 
-More information coming soon!
+### Install a dbt Docker image from Github Packages
+
+Official dbt Docker images are hosted on our [GitHub packages page](https://github.com/orgs/dbt-labs/packages?visibility=public).  We maintain images and tags for each database adapter as well as a `latest` and `<major version>.<minor version>.latest` tag.  They can be installed by utilizing the `docker pull` command:
+```
+docker pull ghcr.io/dbt-labs/<db_adapter_name>:<version_tag>
+```
+
+### Running a dbt Docker image in a container:
+The `ENTRYPOINT` for dbt Docker images is the command `dbt` so you can bind-mount your project to `/usr/app` and use dbt as normal:
```

Comment:
@iknox-fa @jtcohen6 I got stuck here. I downloaded Docker and tried to copy/past the following into a terminal window to no avail. Am I supposed to have a container running first? Does the code in the example have text I need to change? Thanks for your help! 🙇🏻‍♀️

Reply:
@iknox-fa This references `my-dbt`, which seems to depend on having already built an image named "my-dbt" in the example below. Does that sound right? Would it make sense to re-order these?

Reply:
@runleonarun 
If you want to set up a time to walk through how to do this feel free to put something on my calendar.

Reply:
Sounds great!